Flow.Launcher/Converters/BoolToVisibilityConverter.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+
+namespace Flow.Launcher.Converters
+{
+    public class BoolToVisibilityConverter : IValueConverter
```

Comment:
I remember these's a buildin boolconverter (used by @Sparrkle before). So let's use that.

Reply:
I remember I saw it. but where is it?